platform.txt

```diff
@@ -63,7 +63,7 @@ defines.variant={build.defs}
 
 
 ## Compiler and Toolchain
-compiler.path={runtime.tools.arm-none-eabi-gcc-7-2017q4.path}/bin
+compiler.path={runtime.tools.arm-none-eabi-gcc-8-2018-q4-major.path}/bin
```

Comment:
Is this change just snuck in here or is this related somehow?

Reply:
Ahh - good catch. I tried to bypass our new SOP and pushed that commit directly to master. Now I must have based this PR off of the master branch I guess, so this change appeared as a difference on release-candidate. Fixing